src/audio/copier/copier.c

```diff
@@ -404,10 +404,17 @@ static int create_dai(struct comp_dev *parent_dev, struct copier_data *cd,
 		break;
 	case ipc4_i2s_link_output_class:
 	case ipc4_i2s_link_input_class:
-		dai_index[dai_count - 1] = (dai_index[dai_count - 1] >> 4) & 0xF;
-		dai.type = SOF_DAI_INTEL_SSP;
-		dai.is_config_blob = true;
-		type = ipc4_gtw_ssp;
+		{
+			dai_index[dai_count - 1] = (dai_index[dai_count - 1] >> 4) & 0xF;
+			dai.type = SOF_DAI_INTEL_SSP;
+			dai.is_config_blob = true;
+			type = ipc4_gtw_ssp;
+			int ret = ipc4_find_dma_config(&dai, (uint8_t *)copier->gtw_cfg.config_data,
```

Comment:
nit-pick: I would declare 'ret' at the function scope level and remove the braces, this would be nicer with no indentation change.

Reply:
fixed